client/src/TenderList.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+
+const TendersList = () => {
+	const [tenders, setTenders] = useState([]);
+	const [loading, setLoading] = useState(true);
+	const [page, setPage] = useState(1);
+	const [hasMore, setHasMore] = useState(true);
+	const [error, setError] = useState(null);
+
+	const fetchTenders = async (pageNumber) => {
+		setLoading(true);
+		try {
+			const response = await fetch(`/api/tenders?page=${pageNumber}`);
+			if (!response.ok) {
+				throw new Error(`HTTP error! status: ${response.status}`);
+			}
+			const data = await response.json();
+
+			console.log("Fetched data:", data);
+
+			if (Array.isArray(data.tenders)) {
+				setTenders((prevTenders) => [...prevTenders, ...data.tenders]);
```

Comment:
Are we just appending tenders in the same page?

Reply:
after 25 items it should display the second page however I have not succeeded in doing that yet